x/gobenchdata-web/web/app.js

```diff
@@ -46,6 +47,8 @@ export async function generateCharts({
     err.innerText = e;
   }
 
+  perBenchmark = (perBenchmark == 'true');
```

Comment:
can we do this casting to `bool` *before* the call to `generateCharts`, ie [here](https://github.com/bobheadxi/gobenchdata/pull/30/files#diff-80942047d05faac5051c08cc9669c039R73)?